generate/beat/{{cookiecutter.beat}}/dev-tools/README.md

```diff
@@ -0,0 +1,10 @@
+Available scripts
```

Comment:
I can remove `set_version` / `get_version` and `README.md` in `<mybeat>/dev-tools/`

For customization purpose, the *optional* following files overrides the files in libbeat:
* `<mybeat>/dev-tools/packer/readme.md.j2`
* `<mybeat>/dev-tools/packer/version.yml`

Or do you have a nicer implementation in mind?